third_party/conan/configs/install.ps1

```diff
@@ -40,6 +40,12 @@ if (Test-Path Env:ORBIT_OVERRIDE_ARTIFACTORY_URL) {
     if ($process.ExitCode -ne 0) { Throw "Error while adding conan remote." }
 
     conan_disable_public_remotes
+
+    if (Test-Path -Path "$PSScriptRoot\windows_ci") {
+      Write-Host "Found CI-specific Conan config. Installing that as well..."
+      $process = Start-Process $conan.Path -Wait -NoNewWindow -ErrorAction Stop -PassThru -ArgumentList "config","install","$PSScriptRoot\windows_ci"
```

Comment:
```suggestion
      $process = Start-Process $conan.Path -Wait -NoNewWindow -ErrorAction Stop -PassThru -ArgumentList "config", "install", "$PSScriptRoot\windows_ci"
```

If that is possible (not sure if the syntax prevents you from adding the space).

Reply:
I think it's possible. Will do that.

Reply:
Done.